prompt-note/task/index.js

```diff
@@ -0,0 +1,79 @@
+const prompt = require("prompt-sync")();
+
+let count = 0;
+let listItem = [];
+const resultMessage = () => console.log("종료 되었습니다.");
+
+// 질문
+const question = () => {
+  const message = prompt(
+    "1. 전체메모 조회 / 2. 메모 입력 / 3. 메모 모두 삭제하기 / 4. 메모 수정하기 / 0. 종료 : "
+  );
+  const feedback = Number(message);
+
+  if (feedback === 1) {
+    list();
+  } else if (feedback === 2) {
+    write();
+  } else if (feedback === 3) {
+    remove();
+  } else if (feedback === 4) {
+    update();
+  } else {
+  }
+};
+
+// 뒤로가기
+const back = () => {
+  const message = prompt("0. 뒤로가기 / 1: 종료 : ");
+  const feedback = Number(message);
+
+  if (feedback === 0) {
+    question();
+  } else {
+  }
+};
+
+// 전체메모 조회
+const list = () => {
+  if (listItem.length === 0) {
```

Comment:
```suggestion
  if (!listItem.length) {
```
`=== 0` 이 명확하긴 한데 앞에 !붙이는걸로도 가능합니당